src/js/components/ConfigurationMapSection.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const ConfigurationMapSection = (props) => {
+  return (
+    <div className="configuration-map-section pod flush-top flush-right flush-left">
```

Comment:
Yeah, I'll do that! I considered doing it at the time, but opted against it only because the implementer of the `ConfigurationMapSection` wouldn't even have the ability to change these classes. But I do agree it would be neater if all the styles were maintained by the `configuration-map-section` class.